applet/src/menu.c

```diff
@@ -134,23 +134,49 @@ typedef struct menu_mem_base_type menu_mem_base_t ;
 
 extern menu_mem_base_t md380_menu_mem_base[];
 
-void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int enabled) 
+void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int item_count) 
```

Comment:
f = timeout ... see void create_menu_entry_promtg_enable_screen(void) {

  md380_create_menu_entry( md380_menu_id, wt_enable, md380_menu_entry_back+1, md380_menu_entry_back+1, 6, 2 , 1);

after 2 sec call the green function